src/table/create.rs

```diff
@@ -36,23 +37,23 @@ use crate::{
 ///     [
 ///         r#"CREATE TABLE IF NOT EXISTS `character` ("#,
 ///             r#"`id` int NOT NULL AUTO_INCREMENT PRIMARY KEY,"#,
-///             r#"`font_size` int NOT NULL,"#,
+///             r#"`font_size` int NOT NULL COMMENT 'font size',"#,
 ///             r#"`character` varchar(255) NOT NULL,"#,
 ///             r#"`size_w` int NOT NULL,"#,
 ///             r#"`size_h` int NOT NULL,"#,
 ///             r#"`font_id` int DEFAULT NULL,"#,
 ///             r#"CONSTRAINT `FK_2e303c3a712662f1fc2a4d0aad6`"#,
 ///                 r#"FOREIGN KEY (`font_id`) REFERENCES `font` (`id`)"#,
 ///                 r#"ON DELETE CASCADE ON UPDATE CASCADE"#,
-///         r#")"#,
+///         r#") COMMENT 'table comment' "#,
 ///     ].join(" ")
 /// );
 /// assert_eq!(
 ///     table.to_string(PostgresQueryBuilder),
 ///     [
 ///         r#"CREATE TABLE IF NOT EXISTS "character" ("#,
 ///             r#""id" serial NOT NULL PRIMARY KEY,"#,
-///             r#""font_size" integer NOT NULL,"#,
+///             r#""font_size" integer NOT NULL ,"#,
```

Comment:
The extra space comes from 

https://github.com/SeaQL/sea-query/blob/42465d36c5a1895662a33c4ca087347639473632/src/backend/sqlite/table.rs#L24-L25

I'd suggest moving `write!(sql, " ").unwrap();` line into the `prepare_column_spec` method.